packages/wallet-sdk/src/core/rpc/wallet_sendPreparedCalls.ts

```diff
@@ -0,0 +1,51 @@
+import { Hex } from 'viem';
+
+export type WebauthnSignatureType = {
+  type: 'webauthn';
+  data: {
+    /**
+     * The signature is the stringified JSON of the webauthn signature
+     * @example
+     * ```json
+     * {
+     *   "id": "string",
+     *   "rawId": "string",
+     *   "response": {
+     *     "authenticatorData": "string",
+     *     "clientDataJSON": "string",
+     *     "signature": "string"
+     *   },
+     *   "type": "string"
+     * }
+     * ```
+     */
+    signature: string;
+    publicKey: Hex;
+  };
+};
+
+export type Secp256k1SignatureType = {
+  type: 'secp256k1';
+  data: {
+    address: Hex;
+    signature: Hex;
+  };
+};
+
+export type SendPreparedCallsParams = [
+  {
+    version: string;
+    type: string;
+    data: any;
+    chainId: Hex;
+    signature: WebauthnSignatureType | Secp256k1SignatureType;
+  },
+];
+
+export type SendPreparedCallsReturnValue = string[];
+
+export type SendPreparedCallsSchema = {
```

Comment:
I like having the RPC schema. In a follow up we should update `wallet_addSubAccount` and `wallet_connect`. I'll make a task to track it in linear.